test/testSetup.ts

```diff
@@ -243,7 +241,7 @@ export class TestSetup {
     );
   }
 
-  private async createDefaultScratchOrg(): Promise<void> {
+  public async createDefaultScratchOrg(): Promise<void> {
```

Comment:
Why is this changed from private to public?  The function is not called anywhere outside of testSetup.ts.

Reply:
I used it somewhere when I was trying some things and I guess I forgot to change it back to private. Good catch, thanks!